sms-checker/message.py

```diff
@@ -0,0 +1,70 @@
+'''This script will message the subscribers if they require notification'''
+import logging
+from os import environ as ENV
+from dotenv import load_dotenv
+from boto3 import client
+
+def send_sms(subscribers: list[dict]) -> None:
+    '''Uses boto3 to send messages to subscribers by sms'''
+    sms = get_client('sns')
+    logging.info('Client established for SNS')
+    for sub in subscribers:
+        if sub['body']:
+            message = 'STARWATCH ALERT'
+            message += f'\n\n Hello {sub['user'].capitalize()},\n\n'
+            if not sub['at']:
+                message += f"{sub['body'].capitalize()} will be visible tonight in your county!"
+            else:
+                message += f"\n{sub['body'].capitalize()} will be visible tonight"
+                message += f"at {sub['at'].strftime('%H:%M')} in your county!"
+            message += "\n Go to our dashboard for more information."
+            logging.info('Message constructed successfully')
+            response = sms.publish(PhoneNumber=sub['phone'],
+                        Message=message)
+            logging.info('Message sent for subscriber: %s',sub['user'])
+            logging.info('SNS response received: %s', response)
+        else:
+            logging.warning('Invalid data in: %s', sub)
```

Comment:
The function name and how our architecture diagram is currently laid our uses AWS SNS not AWS SMS. We could use a combination of both SMS and SES instead of SNS, but that would be different to what we originally planned. Messages are great though.

Reply:
This will be resolved in a future pull request